src/main/scala/beam/utils/map/R5MapStatsCalculator.scala

```diff
@@ -1,19 +1,21 @@
 package beam.utils.map
 
-import com.conveyal.osmlib.OSM
+import com.conveyal.osmlib.{OSM, Way}
 import com.conveyal.r5.point_to_point.builder.TNBuilderConfig
 import com.conveyal.r5.streets.EdgeStore
 import com.conveyal.r5.streets.EdgeStore.EdgeFlag
 import com.conveyal.r5.transit.TransportNetwork
 
 import scala.collection.JavaConverters._
+import scala.collection.mutable
 import scala.util.Try
 
 object R5MapStatsCalculator {
 
   def main(args: Array[String]): Unit = {
-//    LoggerFactory.getLogger("com.conveyal").asInstanceOf[Logger].setLevel(Level.INFO)
-    val pathToOsm = args(0)
+    //    LoggerFactory.getLogger("com.conveyal").asInstanceOf[Logger].setLevel(Level.INFO)
+    //val pathToOsm = args(0)
```

Comment:
This seems like an odd reversion (similar to my other comments about absolute paths)